app/assets/stylesheets/objects/_grid.scss

```diff
@@ -1,3 +1,7 @@
 .app-grid-column--end {
   text-align: right;
 }
+
+.govuk-width-container {
+  max-width: 1200px;
```

Comment:
As discussed, if we're going to go with a new breaking point we should do it by defining it in `media-queries`. The rest of the code is fine, but we should discuss this with the rest of the team and understand implications.